libraries/AP_AdvancedFailsafe/AP_AdvancedFailsafe.cpp

```diff
@@ -145,6 +145,19 @@ const AP_Param::GroupInfo AP_AdvancedFailsafe::var_info[] = {
     // @Units: s
     AP_GROUPINFO("RC_FAIL_TIME",   19, AP_AdvancedFailsafe, _rc_fail_time_seconds,    0),
 
+    // @Param: TERM_DLY
+    // @DisplayName: Termination Delay
+    // @Description: This is the time in seconds the aircraft will stay at a failsafe waypoint before aero termination.
+    // @User: Advanced
+    // @Units: seconds 
+    AP_GROUPINFO("TERM_DLY",   20, AP_AdvancedFailsafe, _terminal_delay,   30),
```

Comment:
Default this to 0, rather then changing the existing behavior.

Reply:
Thank you your input.  Removed the TERM_ENABLE parameter and set default for "TERM_DLY" to 0.